includes/class-wc-order.php

```diff
@@ -1731,6 +1731,8 @@ public function add_order_note( $note, $is_customer_note = 0, $added_by_user = f
 			);
 		}
 
+		do_action( 'woocommerce_order_note_added', $comment_id, $this );
```

Comment:
Please include a docblock like:

```php
/**
 * Action hook fired when an order note is added.
 *
 * @since 4.4.0
 * @param int      $order_note_id Order note ID.
 * @param WC_Order $order         Order data.
 */
do_action( 'woocommerce_order_note_added', $order_note_id, $this );
```

Reply:
Added in 25fbe5887298ce4c649c4310b4825170f30f9912